imap_data_access/file_validation.py

```diff
@@ -477,6 +477,7 @@ def is_valid_cr(input_cr: str) -> bool:
     "tm": "metakernel",
     "sff": "thruster",
     "L1_de": "lagrange_point",
+    "bcp": "high_precision_planetary_constants",
```

Comment:
This is for `earth_latest_high_prec.bpc` correct? .bpc files are generically called planetary CK files or to fit with our other file names, perhaps we should use planetary_attitude, or more specifically earth_attitude.
```suggestion
    "bpc": "earth_attitude",
```